src/lib/down-arrow.svelte

```diff
@@ -1,5 +1,18 @@
+<script>
+	export let onClickAction = () => {};
```

Comment:
Maybe it's OK to not define this empty function here and keep it undefined. Wasn't really thinking too much about this part when I started it.